scripts/docker-integration-tests/query_fanout/test.sh

```diff
@@ -5,6 +5,7 @@ set -xe
 source $GOPATH/src/github.com/m3db/m3/scripts/docker-integration-tests/common.sh
 REVISION=$(git rev-parse HEAD)
 COMPOSE_FILE=$GOPATH/src/github.com/m3db/m3/scripts/docker-integration-tests/query_fanout/docker-compose.yml
+WARNING_TESTS=$GOPATH/src/github.com/m3db/m3/scripts/docker-integration-tests/query_fanout/warning.sh
```

Comment:
Is it worth just exporting it as a function, similar to `source .../common.sh` and then calling something like `warning_tests`